Apps/W1/Shopify/app/src/Companies/Enums/ShpfyCompanyMapping.Enum.al

```diff
@@ -3,25 +3,28 @@ namespace Microsoft.Integration.Shopify;
 /// <summary>
 /// Enum Shpfy Company Mapping (ID 30151) implements Interface Shpfy ICompany Mapping.
 /// </summary>
-enum 30151 "Shpfy Company Mapping" implements "Shpfy ICompany Mapping", "Shpfy ICustomer/Company Mapping"
+enum 30151 "Shpfy Company Mapping" implements "Shpfy ICompany Mapping", "Shpfy IFind Company Mapping"
```

Comment:
Do you need to add this additional interface here? I though it would work without it?